### PR TITLE
Preparation for next release

### PR DESCRIPTION
**Pull request by rete**
BEGINRELEASENOTES
- v02-01 scripts:
    - Set iLCSoft version to v02-01

ENDRELEASENOTES

### CODE REVIEW

File: scripts/use_gcc82_cvmfs_el7.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#---- use gcc and python from SFT in cvmfs
+source /cvmfs/sft.cern.ch/lcg/releases/gcc/8.2.0/x86_64-centos7/setup.sh
+export PATH=/cvmfs/sft.cern.ch/lcg/releases/LCG_96/Python/2.7.16/x86_64-centos7-gcc8-opt/bin:$PATH
+export LD_LIBRARY_PATH=/cvmfs/sft.cern.ch/lcg/releases/LCG_96/Python/2.7.16/x86_64-centos7-gcc8-opt/lib:$LD_LIBRARY_PATH
+
+# --- use correct tbb for ROOT
+export LD_LIBRARY_PATH=/cvmfs/sft.cern.ch/lcg/releases/tbb/2019_U1-5939b/x86_64-centos7-gcc8-opt/lib/:$LD_LIBRARY_PATH
```
Comment (rete) [inline scripts/use_gcc82_cvmfs_el7.sh]: Why is this needed ? ROOT compiles TBB internally and is not taking it from CVMFS

**Comment by gaede** [comment]
Rebased this branch with master to include #84.

**Comment by rete** [comment]
I just rebased on master. Most of the commits made in this branch are already included in the master

**Comment by rete** [comment]
@gaede I'm done with all the versioning. Remains the ILDConfig tag, but there I need inputs from Yasser (refitting processors) and the generator group (expected background).
